wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/button/CommandGenericHID.java

```diff
@@ -260,6 +261,37 @@ public Trigger axisGreaterThan(int axis, double threshold, EventLoop loop) {
         Pair.of(axis, threshold), k -> new Trigger(loop, () -> getRawAxis(axis) > threshold));
   }
 
+  /**
+   * Constructs a Trigger instance that is true when the axis value is active (non-zero) given a
+   * {@code deadband}, attached to the given loop.
+   *
+   * @param axis The axis to read, starting at 0
+   * @param deadband The value used to deadband the axis value. The trigger returns true if the
+   *     deadbanded value is non-zero.
+   * @param loop the event loop instance to attach the trigger to.
+   * @return a Trigger instance that is true when the deadbanded axis value is active (non-zero).
```

Comment:
I would rephrase this to be similar to `axisGreaterThan`/`axisLessThan`, since this is equivalent to `axisGreaterThan(x, y).or(axisLessThan(x, -y))`. Use threshold instead of deadband, and describe when the trigger is active relative to threshold

Reply:
AxisMagGreaterThan?

Reply:
So I renamed it to axisMagnitudeGreater and cleaned up the comments.